mvnw

```diff
@@ -0,0 +1,322 @@
+#!/bin/sh
```

Comment:
As this repository includes third party content we need to create a attribution document (usually named THIRD-PARTY-LICENSES, THIRD-PARTY-NOTICES, or just THIRD-PARTY) to comply with the license conditions of that third-party content.

Please check an example here: https://github.com/aws-samples/aws-iot-edukit-examples/blob/main/THIRD-PARTY-LICENSES.txt

For used as part of package manager config files (package.json, requirements.txt, pom.xml etc) it is not necessary to add add them to the attribution file.

Reply:
Maven wrapper removed in e9d4fff, hence no THIRD PARTY file